docs/src/Guides/20 Scales.md

```diff
@@ -203,6 +203,36 @@ bot.grow_scale("Filename_here")
 bot.start("token")
 ```
 
-Finally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
+Additionally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
 To do this, simply run `bot.regrow_scale("Filename_here")` and your new code will be used. Bare in mind any tasks your scale
 is doing will be abruptly stopped.
+
+
+Finally, you can also pass any **keyword** arguments into module's `setup` and `teardown` functions, it will also send them into your `Scale.shed` method.
+
+Here is a basic "Scale switching" example:
+
+```python
+from dis_snek import Scale
+
+class SomeScale(Scale):
+    def __init__(self, bot, some_arg: int = 0):
+        ...
+
+class AnotherScale(Scale):
+    def __init__(self, _, another_arg: float = 0.0):
```

Comment:
Why are you using `_` instead of bot here?

Reply:
according to pep this would actually be correct, seeing as you dont use the argument, its purely there because `__new__` requires it to be passed.

Reply:
For the sake of a readable example, we should definitely use the name.